### ISSUE TITLE
Please come into compliance with the license of my font

### ISSUE THREAD

**Issue by ctrlcctrlv** [Ryo-N7/tvthemes#14]
Hello,

I am the author of Some Time Later.

By redistributing it without crediting me, you are in violation of its license.

Please come into compliance with the license or else cease unauthorized distribution of my font.

You can come into compliance simply by copying my font's LICENSE file into the appropriate directory, and crediting me for it in README.md at your option. You should check that the other fonts you are using have their licenses attached if necessary.

**Comment by Ryo-N7** [comment]
Hi,

Sorry about this. I've added the license here: https://github.com/Ryo-N7/tvthemes/tree/master/inst/fonts/SpongeBob

I had credited you in the documentation for the function that imports the font from the start:
![font](https://user-images.githubusercontent.com/19657164/99150677-879e7680-26d9-11eb-9789-863539a0f43f.PNG)

but I realize that's not enough and have now also listed you in the credits in the README.

Sorry again and thanks for the awesome font!
